pkg/util/conditions/conditions.go

```diff
@@ -0,0 +1,46 @@
+package annotations
```

Comment:
Guess, this should be re-named to `conditions` :)

Reply:
My bad. Fixed this and some other annotation references.